securedrop_client/logic.py

```diff
@@ -432,6 +434,9 @@ def login_offline_mode(self):
         Allow user to view in offline mode without authentication.
         """
         self.gui.hide_login()
+        # Clear clipboard contents in case of previously pasted creds (user
+        # may have attempted online mode login, then switched to offline)
+        self.gui.clear_clipboard()
```

Comment:
The duplication here is intentional; I felt that this behavior should not be a side effect of another function, but explicitly invoked so the intended business logic is clear.